public/css/post-session.css

```diff
@@ -0,0 +1,4 @@
+*,html {
```

Comment:
Yeah you do not want to be doing this to your html tag.

Put each page in a wrapper
```html
<div id=post-session-wrapper>
  <!--everything else -->
</div>
```